src/main/java/dev/slimevr/vr/MountingCalibration.java

```diff
@@ -0,0 +1,154 @@
+package dev.slimevr.vr;
+import io.eiren.util.collections.FastList;
+import io.eiren.util.logging.LogManager;
+import dev.slimevr.VRServer;
+import dev.slimevr.vr.trackers.IMUTracker;
+import dev.slimevr.vr.trackers.ReferenceAdjustedTracker;
+import dev.slimevr.vr.trackers.Tracker;
+import dev.slimevr.vr.trackers.TrackerConfig;
+
+import com.jme3.math.FastMath;
+import com.jme3.math.Quaternion;
+
+public class MountingCalibration {
+
+	private final VRServer server;
+	private FastList<Quaternion> trackerOrientationIdle = new FastList<>();
+	private FastList<Tracker> allTrackers = new FastList<>();
+	private int trackerNumber;
+	private static float TENTH_THIRD = 0.03333f, FIFTH_THIRD = 0.06666f;
+	private static float VERTICAL_OFFSET = 0.7f;
+
+	public MountingCalibration(VRServer server){
+		this.server = server;
+	}
+	public void SetIMUMountingRotation(float rad, IMUTracker imu, Tracker t){ // Sets the mounting rotation (rad) of a given imu tracker and saves it.
+		imu.setMountingRotation(rad);
+		TrackerConfig tc = server.getTrackerConfig(t);
+		imu.saveConfig(tc);
+		server.saveConfig();
+	}
+	public void GetIdle(){ // Gets the orientation of the first pose for each tracker one by one.
+		allTrackers = (FastList<Tracker>) server.getAllTrackers();
```

Comment:
Don't cast to specific type, the type of returned varible may change in the future, you're shooting yourself in the leg. You also don't need `allTrackers` to be a member at all, nor for it to be FastList specifically. This doesn't get called all that often, and getAllTrackers() returns new object, why are you storing it at all?